src/Models/BaseElement.php

```diff
@@ -477,7 +477,7 @@ public function getController()
             return $this->controller;
         }
 
-        $controllerClass = self::config()->controller_class;
+        $controllerClass = static::config()->controller_class;
```

Comment:
This should have been static previously, though self also works seemingly untintuitively. BaseElement::config() behaves differently from BaseElement::config()